src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -417,8 +417,12 @@ def _to_vespa_tensor_query(self, marqo_query: MarqoTensorQuery) -> Dict[str, Any
         select_attributes = self._get_select_attributes(marqo_query)
         summary = common.SUMMARY_ALL_VECTOR if marqo_query.expose_facets else common.SUMMARY_ALL_NON_VECTOR
         score_modifiers = self._get_score_modifiers(marqo_query)
-        ranking = common.RANK_PROFILE_EMBEDDING_SIMILARITY_MODIFIERS if score_modifiers \
-            else common.RANK_PROFILE_EMBEDDING_SIMILARITY
+
+        if self._marqo_index_version < self._VERSION_2_10_0:
```

Comment:
Do we cover this in integration tests? We'll need to create indices with older version of marqo, right?

Reply:
For now, we just need to do manual backwards compatibility tests, as I don't think this is supported in integration tests yet

Reply:
Requires backwards compatibility tests. We can't test this in simple integ tests